pkg/runners/builtin_runner/range_check.go

```diff
@@ -0,0 +1,72 @@
+package builtinrunner
+
+import (
+	"errors"
+	"math"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+var ErrRangeOutOfBounds = errors.New("RangeCheckNumOutOfBounds")
```

Comment:
Lets use a more descriptive error message, like "Range check validation failed, value  is out of valid range"
We can also include  the value and the upper bound